### PR TITLE
dojson : remove punctuation for work_access_point

### PR DESCRIPTION
**Pull request by rerowep**
Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

### CODE REVIEW

File: rero_ils/dojson/utils.py [hunk 1]
```diff
@@ -1763,11 +1763,20 @@ def update_part(self, value_data, subfield_code, subfield_data):
         :param subfield_data: part number or name depending of `subfield_code`
         :type subfield_data: str
         """
+        def remove_last_dot(value):
+            """Remoces last dot from value."""
```
Comment (Garfield-fr) [inline rero_ils/dojson/utils.py:1767]: `Removes`

File: rero_ils/dojson/utils.py [hunk 2]
```diff
@@ -1763,11 +1763,20 @@ def update_part(self, value_data, subfield_code, subfield_data):
...
+            if value.count('.') == 1:
+                value = value.rstrip('.')
+            return value
```
Comment (Garfield-fr) [inline rero_ils/dojson/utils.py:1770]: maybe
```
if value[-1] == '.':
  return value[:-1]
```

File: rero_ils/modules/documents/dojson/contrib/marc21tojson/utils.py [hunk 3]
```diff
@@ -520,7 +520,7 @@ def build_agent(marc21, key, value):
         name = not_repetitive(
             marc21.bib_id, marc21.bib_id, key, value, 'a')
         agent_data['preferred_name'] = remove_trailing_punctuation(
-            name)
+            name, )
```
Comment (Garfield-fr) [inline rero_ils/modules/documents/dojson/contrib/marc21tojson/utils.py:523]: why add `,` ?

**Review by Garfield-fr — APPROVED** [review]

File: rero_ils/dojson/utils.py (hunk 1 above)
Comment (rerowep) [reply rero_ils/dojson/utils.py:1767]: Removes last dot from value if there are no other dots.

File: rero_ils/dojson/utils.py (hunk 2 above)
Comment (zannkukai) [reply rero_ils/dojson/utils.py:1770]: @Garfield-fr This is not the purpose of this function if you read the docstring ;-)

**Review by zannkukai — APPROVED** [review]

**Review by vgranata — APPROVED** [review]

**Pull request merged**